awx/ui_next/src/components/HorizontalSeparator/HorizontalSeparator.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
PatternFly recently promoted the [Divider component](https://github.com/patternfly/patternfly-react/commits/master/packages/patternfly-4/react-core/src/components/Divider/Divider.tsx) from experimental. Just wanted to put this on your radar.